libcontainer/cgroups/fs2/create.go

```diff
@@ -0,0 +1,103 @@
+package fs2
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/opencontainers/runc/libcontainer/configs"
+)
+
+func neededControllers(cgroup *configs.Cgroup) []string {
```

Comment:
Please add godoc that the list doesn't contain "pseudo" controllers, i.e. devices and freezer

Reply:
done